mockups/tokens.css

```
@@ -0,0 +1,247 @@
+  --risk-liquid: #60a5fa;
+
+  /* Typography — styles.css @theme */
+  --text-nano: 10px;
+  --text-micro: 11px;
+  --text-hint: 12.5px;
+  --text-control: 13px;
+  --text-title: 22px;
+  --radius-sm: 10px;
+  --radius-md: 12px;
+  --radius-lg: 16px;
+  --font-body: "Manrope", system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)